### ISSUE TITLE
[CAL-3652] bi-directional sync for outlook

### ISSUE THREAD

**Issue by PeerRich** [calcom/cal.com#14913]
tldr: when making changes manually inside the outlook calendar, it should reflect inside cal.com

1. I set up the meeting types and don't have any limitations on the meeting other than a 15 minute buffer on each side.
2. I did a test booking on a meeting time a week out and the workflow all worked. I saw the meeting appear on my calendar and got all the confirmation stuff I setup in the workflow.
3. I then went into my Outlook and canceled the meeting. It disappeared from my calendar, as it should.
4. I went into the Cal.com link to try to book another as a test and noticed the time I just canceled was still showing as booked (not an option). I refreshed a few times and even waited a few hours and it still shows up as not available even though it is on my calendar. I tried it a second time with another time and followed the same steps and it also doesn't appear after cancelation.
5. I checked all my settings and I don't see anything I have that would limit this. I can still book other times on my calendar.

<sub>[CAL-3652](https://linear.app/calcom/issue/CAL-3652/bi-directional-sync-for-outlook)</sub>
